app/src/main/java/com/courseproject/tindar/MainActivity.java

```diff
@@ -1,24 +1,88 @@
 package com.courseproject.tindar;
 
-import android.content.Intent;
+
 import android.os.Bundle;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.EditText;
+
+import android.content.Intent;
+import android.view.Gravity;
+import android.view.MotionEvent;
 import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.PopupWindow;
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import com.courseproject.tindar.controllers.login.LoginController;
+import com.courseproject.tindar.ds.DatabaseHelper;
+import com.courseproject.tindar.usecases.login.LoginDsGateway;
+import com.courseproject.tindar.usecases.login.LoginInteractor;
+
 public class MainActivity extends AppCompatActivity {
```

Comment:
should probably refactor this at some point such that MainActivity opens other activities depending on existing info (e.g. properly authenticated?) rather than directly sending you to the log in / sign up page 
#78